### ISSUE TITLE
Uncaught TypeError: filterFunc is not a function(TreeSelect)

### ISSUE THREAD

**Issue by tancgo** [ant-design/ant-design#19523]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/tree-select-onsearch-f05uf)

### Steps to reproduce
- search keywords in select-tree
- chrome console error :  

```javascript
util.js:262 Uncaught TypeError: filterFunc is not a function
    at mapFilteredNodeToData (util.js:262)
    at Array.map (<anonymous>)
    at getFilterTree (util.js:279)
    at Select.js:616
    at HTMLUnknownElement.callCallback (react-dom.development.js:147)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:196)
    at invokeGuardedCallback (react-dom.development.js:250)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:265)
    at executeDispatch (react-dom.development.js:571)
    at executeDispatchesInOrder (react-dom.development.js:596)
    at executeDispatchesAndRelease (react-dom.development.js:695)
    at executeDispatchesAndReleaseTopLevel (react-dom.development.js:704)
    at Array.forEach (<anonymous>)
    at forEachAccumulated (react-dom.development.js:674)
    at runEventsInBatch (react-dom.development.js:844)
    at runExtractedEventsInBatch (react-dom.development.js:852)
    at handleTopLevel (react-dom.development.js:5030)
    at batchedUpdates$1 (react-dom.development.js:21425)
    at batchedUpdates (react-dom.development.js:2247)
    at dispatchEvent (react-dom.development.js:5110)
    at react-dom.development.js:21482
    at Object.unstable_runWithPriority (scheduler.development.js:255)
    at interactiveUpdates$1 (react-dom.development.js:21481)
    at interactiveUpdates (react-dom.development.js:2268)
    at dispatchInteractiveEvent (react-dom.development.js:5086)
```

### What is expected?
 correct search results with no console error 

### What is actually happening?
correct search results with console error 

| Environment | Info |
|---|---|
| antd | 3.24.3 |
| React | latest |
| System | windows |
| Browser | chrome 77.0.3865.90 |

**Comment by afc163** [comment]
It works fine in your codesandbox.

![](http://g.recordit.co/PxT5YywsrM.gif)

**Comment by tancgo** [comment]
> It works fine in your codesandbox.
> 
> ![](https://camo.githubusercontent.com/82892959d2e50fe14ca01a063d50ca2c7f690912/687474703a2f2f672e7265636f726469742e636f2f5078543559797773724d2e676966)

**Comment by tancgo** [comment]
> > It works fine in your codesandbox.
> > ![](https://camo.githubusercontent.com/82892959d2e50fe14ca01a063d50ca2c7f690912/687474703a2f2f672e7265636f726469742e636f2f5078543559797773724d2e676966)

![image](https://user-images.githubusercontent.com/19163130/68006420-7c85cd80-fcb3-11e9-97cc-4323b22ddf33.png)

it will appear at chrome console , and it will also works err in my  real project

**Comment by yoyo837** [comment]
PR send https://github.com/react-component/tree-select/pull/203

**Comment by afc163** [comment]
Fixed in rc-tree-select@2.9.4
